build.sbt

```diff
@@ -236,9 +236,21 @@ lazy val scaladocSettings = Seq(
 lazy val scoverageSettings = Seq(
   coverageMinimum := 60,
   coverageFailOnMinimum := false,
-  coverageHighlighting := true
+  coverageHighlighting := true,
+  coverageEnabled := {
+    if (is_2_12(scalaVersion.value))
+      coverageEnabled.value
+    else
+      false
+  }
 )
 
+def is_2_12(scalaVersion: String): Boolean =
+  CrossVersion.partialVersion(scalaVersion) match {
+    case Some((2, 12)) => true
+    case Some((2, _))  => false
+  }
```

Comment:
```
/home/travis/build/laserdisc-io/laserdisc/build.sbt:249: warning: match may not be exhaustive.
It would fail on the following inputs: None, Some(_)
  CrossVersion.partialVersion(scalaVersion) match {
                             ^
```
also, what about
```scala
val is211 = Def.setting {
  CrossVersion.partialVersion(scalaVersion.value) match {
    case Some((2, 11)) => true
    case _             => false
}
```
then you can use `coverageEnabled := if (is211.value) false else coverageEnabled.value`

(and propose to reuse for the `refined` different dependency...

wdyt?

Reply:
yep, I was actually going to fix the matcher.

But really good point with the `refined` - completely makes sense. Will reuse the same logic.